apps/lily-wallet/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3'
+
+services:
+  frontend:
+    image: kaybesee/lily-frontend:latest
+    restart: on-failure
+    stdin_open: true
+    build:
+      dockerfile: Dockerfile.frontend
+      context: ./
+    ports:
+      - '42069:80'
+
+  express:
+    privileged: true
+    image: kaybesee/lily-express:latest
+    restart: on-failure
+    volumes:
+      - ${APP_DATA_DIR}/config
+      - ${LND_DATA_DIR}/:/lnd:ro
+      - /dev:/dev
+    build:
+      dockerfile: Dockerfile.express
+      context: ./
+    environment:
+      - EXPRESS_PORT=5000
+      - ELECTRUM_IP=$ELECTRUM_IP
+      - ELECTRUM_PORT=$ELECTRUM_PORT
+      - APP_PASSWORD=$APP_PASSWORD
+      - APP_DATA_DIR=$APP_DATA_DIR
```

Comment:
This comment is a little confusing to me. 

I am passing in `APP_DATA_DIR` in order to save/retrieve the encrypted configuration file ([code](https://github.com/Lily-Technologies/lily-wallet/blob/master/apps/express/src/utils/setInitialConfig.ts#L14)). 

`APP_DATA_DIR` = `/home/umbrel/umbrel/app-data/lily-wallet`

Are you saying I should save it in `/home/umbrel/umbrel/app-data/lily-wallet/data` instead of `/home/umbrel/umbrel/app-data/lily-wallet` and pass in APP_DATA_DIR as ${APP_DATA_DIR}/data ? 

Sorry, my docker ignorance is showing here.